sci-biology/SEECER/SEECER-0.1.3-r3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="SEquence Error Corrector for RNA-Seq reads"
+HOMEPAGE="http://sb.cs.cmu.edu/seecer/"
+SRC_URI="
+	http://sb.cs.cmu.edu/seecer/downloads/"${P}".tar.gz
+	http://sb.cs.cmu.edu/seecer/downloads/manual.pdf -> "${PN}"-manual.pdf"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+# although has bundled jellyfish-1.1.11 copy it just calls the executable during runtime
+# seems jellyfish-2 does not accept same commandline arguments
+DEPEND="
+	sci-libs/gsl:0=
+	sci-biology/seqan:0="
+RDEPEND="${DEPEND}
+	sci-biology/jellyfish:1"
+
+S="${WORKDIR}/${P}/SEECER"
+
+PATCHES=(
+	"${FILESDIR}"/remove-hardcoded-paths.patch
+	"${FILESDIR}"/run_seecer.sh.patch
+	"${FILESDIR}"/run_jellyfish.sh.patch
+	"${FILESDIR}"/${PN}-increase-max-sequence-length.patch
+	"${FILESDIR}"/${PN}-remove-flags.patch
+)
+
+pkg_pretend(){
+	# openmp is a hard requirement due to no gomp switch and source code assuming it is available
+	[[ ${MERGE_TYPE} != "binary" ]] && tc-check-openmp
+}
+
+pkg_setup(){
+	[[ ${MERGE_TYPE} != "binary" ]] && tc-check-openmp
+}
+
+src_prepare(){
+	rm -f bin/.run*.swp || die
+	rm -f src/*.o pipeline/*.o || die
+	rm -rf seqan-cxx || die
```

Comment:
`-f` makes `rm` exit successfully even if the files do not exist, as such it makes the `|| die` useless. Therefore, I have removed the `-f` before merging.